packages/cdk8s-cli/lib/import/crd.ts

```diff
@@ -98,11 +73,48 @@ export class ImportCustomResourceDefinition extends ImportBase {
       schema: this.schema
     });
 
+    code.line();
+    types.generate(code);
+  }
+}
+
+export class ImportCustomResourceDefinition extends ImportBase {
+  public static async match(source: string): Promise<undefined | CustomResourceApiObject[]> {
+    let manifest;
+    if (source.startsWith('https://')) {
+      manifest = yaml.parseAllDocuments(await httpsGet(source));
+    } else if (path.extname(source) === '.yaml' || path.extname(source) === '.yml' || path.extname(source) === '.json') {
+      if (!(await fs.pathExists(source))) {
+        throw new Error(`can't find file ${source}`);
+      }
+
+      manifest = yaml.parseAllDocuments(await fs.readFile(source, 'utf-8'));
+    }
+
+    if (!manifest) {
+      return undefined;
+    }
+
+    return manifest?.map((doc: yaml.ast.Document) => doc.toJSON())
+  }
+
+  private readonly CRDs: CustomResourceDefinition[] = [];
+  
+  constructor(manifest: CustomResourceApiObject[]) {
+    super();
+
+    this.CRDs = manifest?.map((apiObject: CustomResourceApiObject) => new CustomResourceDefinition(apiObject));
+  }
+
+  public get moduleName() {
+    return MODULE_NAME.toLocaleLowerCase();
```

Comment:
I think we should generate a separate file for each CRD with module name based on the kind and not a single file for all of them (as if each CRD was in a separate manifest).

Reply:
Done!